www/hugo.toml

```diff
@@ -19,40 +22,39 @@ author = "authors"
 # category = "categories"
 
 [module]
-    [[module.mounts]]
-        source = "../code"
-        target = "assets/js"
-    [[module.mounts]]
-        source = "assets"
-        target = "assets"
-    [[module.mounts]]
-        source = "../corpus"
-        target = "static/corpus"
-    [[module.mounts]]
-        source = "../layouts"
-        target = "static/layouts"
-    [[module.mounts]]
-        source = "static"
-        target = "static"
+[[module.mounts]]
+source = "../code"
+target = "assets/js"
+[[module.mounts]]
+source = "assets"
+target = "assets"
+[[module.mounts]]
+source = "../corpus"
+target = "static/corpus"
+[[module.mounts]]
+source = "../layouts"
+target = "static/layouts"
+[[module.mounts]]
+source = "static"
+target = "static"
 
 [menus]
     [[menus.main]]
-        name    = "Ergo‑L"
+        name    = "Accueil"
         pageRef = "/"
         weight  = 1
     [[menus.main]]
         name    = "Présentation"
         pageRef = "/presentation"
-        parent  = "Ergo‑L"
-        weight  = 1
+        weight  = 2
     [[menus.main]]
         name    = "Installation"
         pageRef = "/installation"
-        parent  = "Ergo‑L"
+        weight  = 3
```

Comment:
I’m not sure we should alter the menu structure; but if we do, and if we add two more top-level items as you suggest, we’ll definitely have to revamp the whole menu bar.

I’d be in favor of keeping the menu as-is for now and address this in a separate PR.